src/plugins/plugin.colors.ts

```diff
@@ -1,8 +1,9 @@
 import {DoughnutController, PolarAreaController} from '../index.js';
-import type {Chart, ChartConfiguration, ChartDataset} from '../types.js';
+import type {Chart, ChartDataset} from '../types.js';
 
 export interface ColorsPluginOptions {
   enabled?: boolean;
+  forceColorsPlugin: boolean;
```

Comment:
```suggestion
  forceColorsPlugin?: boolean;
```

Reply:
Actually does not make a difference for now since all plugin opts are put through deepPartial so they will always be optional, might be worth changing in V5